src/app/components/recipes-and-filters/recipes-and-filters.component.ts

```diff
@@ -33,4 +41,33 @@ export class RecipesAndFiltersComponent {
     if (filters.limit) { r.limit = filters.limit };
     return r;
   }
+
+  /**
+   * Update the sidenav property for the current window.
+   */
+  updateSidenavForWindow() {
+    if (isPlatformBrowser) {
+      if (window.innerWidth < 768) {
+        this.navMode = 'over';
+        this.sidenav.close();
+      } else {
+        this.sidenav.open();
+      }
+    }
+  }
+
+  /**
+   * Listen to window resize.
+   */
+  @HostListener('window:resize', ['$event'])
```

Comment:
I am pretty sure this is supported with universal but I want to make sure..just asked for some info on some of the universal chats to be sure this is not going to break when we start using universal.